### PR TITLE
[ZEPPELIN-1930](HotFix) PythonInterpreter syntax error.

### PR DESCRIPTION
**Pull request by astroshim**
### What is this PR for?
This PR fixes syntax error of PythonInterpreter.

### What type of PR is it?
Bug Fix

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-1930

### How should this be tested?
Please run following code on the paragraph of PythonInterpreter.
```
for x in range(0,3):  print ("hi")
```

### Screenshots (if appropriate)
- before
image

- after
image

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

**Comment by Leemoonsoo** [comment]
Thanks for the fix. \cc @agoodm 

How about add an unittest for this case in https://github.com/apache/zeppelin/blob/master/python/src/test/java/org/apache/zeppelin/python/PythonInterpreterTest.java, if it is not too difficult?

**Comment by agoodm** [comment]
I am on vacation right now so I can't thoroughly examine this, but have you confirmed that this isn't a problem for the pyspark interpreter?

**Comment by 1ambda** [comment]
I faced the same problem too. Thanks for fix!

**Comment by astroshim** [comment]
@agoodm pyspark interpreter is working well.
image

**Comment by astroshim** [comment]
@Leemoonsoo I added testcase, could you help to review?

File: python/src/test/java/org/apache/zeppelin/python/PythonInterpreterTest.java [hunk 1]
```diff
@@ -172,6 +181,18 @@ public void testInterpret() {
     assertEquals("%text print a", result.message().get(0).toString());
   }
 
+  @Test
+  public void testInterpret2() {
```
Comment (khalidhuseynov) [inline python/src/test/java/org/apache/zeppelin/python/PythonInterpreterTest.java]: how about a bit more detailed test name here? maybe `testInterpretInvalidSyntax` or something reflecting context more

Comment (astroshim) [inline python/src/test/java/org/apache/zeppelin/python/PythonInterpreterTest.java]: I fixed it. Thanks!

**Comment by khalidhuseynov** [comment]
LGTM!

**Comment by Leemoonsoo** [comment]
LGTM and merge to master if there're no more discussions

**Comment by cloverhearts** [comment]
LGTM!

**Pull request closed without merging**